PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,9 @@
 <!--
 
+All new RFCs must:
+
+* have examples of the proposed behavior, preferably in the form of a script (or scripts) which shows the proposed behavior.
```

Comment:
Pester form is good too.

Reply:
sure - a pester script is certainly a valid script

Reply:
Indeed, but if we explicitly encourage Pester (doesn't necessarily have to be entirely _mandatory_) then we have some tests to work with right from the beginning as either the PS team or community members get to work on implementing the proposal. 🙂

Reply:
I am somewhat worried about only pester tests. Pester tests don't generally test formatting, which I believe is an important aspect of new behaviors. It's important to _see_ the results of a new cmdlet. To date, the number of examples in the RFCs has been woefully low (hardly any), and it makes judging new behavior extremely difficult.